Source: Timo Gurr <tgurr@exherbo.org>
Upstream: TODO
Reason: Fix install location of shared data

diff -Naur mediastreamer2-2.16.1/src/CMakeLists.txt mediastreamer2-2.16.1.new/src/CMakeLists.txt
--- mediastreamer2-2.16.1/src/CMakeLists.txt	2017-07-21 15:00:47.000000000 +0200
+++ mediastreamer2-2.16.1.new/src/CMakeLists.txt	2018-05-04 11:17:13.232247242 +0200
@@ -730,7 +730,7 @@
 
 if(ENABLE_VIDEO)
 	install(FILES voip/nowebcamCIF.jpg
-		DESTINATION ${PACKAGE_DATA_DIR}/images
+		DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/images"
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ
 	)
 endif()
